### ISSUE TITLE
Error on start

### ISSUE THREAD

**Issue by skreiker** [michaelrice/WeakSSL#8]
Hello , I have installed your grails plugin and when I start grails I get the following error
2015-03-26 17:35:06,789-0400 [localhost-startStop-1|]() ERROR GrailsAutoTrustModeSSL - Could not read the file /.grails/2.4.4/projects/ems/ssl/keystore

I am using grails 2.4.4
Any idea what could be happening

**Comment by linguistbreaker** [comment]
This directory - ~/.grails/$GRAILS_VERSION/ssl/keystore - is created when you run-app --https . The --https runs grails (embedded jetty or tomcat) in https w/ ssl - it will create the file there with a self signed cert or use whatever file you have in there. 

The error shouldn't be a problem, but you can run with the --https arg or just touch a file there so that the error isn't thrown.

http://stackoverflow.com/questions/2364602/ssl-tomcat-and-grails

TL;DR - run the command : 
touch ~/.grails/$GRAILS_VERSION/projects/$PROJECT_NAME/ssl/keystore

**Comment by cahyocayo** [comment]
how to run this plugin on grails 3.14 ?

**Comment by vsachinv** [comment]
@cahyocayo I have published the plugin for grails 3.x. 
https://bintray.com/sachinverma/plugins/WeakSSL
